crates/ark/src/variables/r_variables.rs

```diff
@@ -357,6 +386,47 @@ impl RVariables {
         }
     }
 
+    /// Gets the value of the special variable '.Last.value' (the value of the
+    /// last expression evaluated at the top level), if show_last_value is true.
+    ///
+    /// For backward compatibility, also checks the global option "positron.show_last_value"
+    /// if it exists, but the instance variable takes precedence.
+    ///
+    /// Returns None in all other cases.
+    fn last_value(&self) -> Option<PositronVariable> {
+        // First check our instance variable
+        let show_last_value = if self.show_last_value {
+            true
```

Comment:
It isn't clear to me why we need `self.show_last_value`, or why the global option is mentioned here as a backwards compatible thing. It feels reasonable to me for that to be the "main feature" not part of backwards compat.